src/app/main.cpp

```diff
@@ -124,11 +127,32 @@ int main(int argc, char *argv[])
     setupDpi();
 #endif
 
+    // we need a copy of the arguments to extend them with the params file contents (see right below)
+    std::vector<char*> arguments;
+    for (int i = 0; i < argc; ++i) {
+        arguments.push_back(::strdup(argv[i]));
+    }
+
+    // If there is <exeName>.params file near the executable, add its contents to the argv array
+    QString paramsFile = Utils::Fs::executablePath() + QLatin1String(".params");
+    if (QFileInfo::exists(paramsFile)) {
+        QFile fn {paramsFile};
+        if (fn.open(QIODevice::ReadOnly)) {
+            for (const auto& s : QString::fromLocal8Bit(fn.readAll().trimmed())
+                    .split(QRegularExpression(QLatin1String(R"([ \t](?=([^"]*"[^"]*")*[^"]*$))")))) {
```

Comment:
Just a suggestion: Maybe make the QRegularExpression a variable so it is easier to read?